src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = {new Bulldozer(),new Truck(),new Excavator()};
```

Comment:
The spaces after commas needed. Please use code formatting in IDE.

Reply:
done